src/App.js

```diff
@@ -3,13 +3,13 @@ import { NotificationsPortal } from '@redhat-cloud-services/frontend-components-
 import { Main } from '@redhat-cloud-services/frontend-components/components/Main';
 import { IntlProvider } from 'react-intl';
 import { BrowserRouter as Router } from 'react-router-dom';
-import { getBaseName } from '@redhat-cloud-services/frontend-components-utilities/files/helpers';
```

Comment:
It might be better to split helpers file into multiple smaller ones.

Reply:
If it would be smaller than 595 bytes... :smile:

Reply:
> It might be better to split helpers file into multiple smaller ones.

I concur